rtgui/gradient.h

```diff
@@ -42,12 +42,11 @@ class Gradient : public ToolParamBlock, public AdjusterListener, public Foldable
     void setDefaults    (const rtengine::procparams::ProcParams* defParams, const ParamsEdited* pedited = NULL);
     void setBatchMode   (bool batchMode);
 
-    void updateGeometry (int centerX_, int centerY_, double feather_, double degree_);
-
     void adjusterChanged (Adjuster* a, double newval);
     void enabledChanged  ();
     void setAdjusterBehavior (bool degreeadd, bool featheradd, bool strengthadd, bool centeradd);
     void trimValues          (rtengine::procparams::ProcParams* pp);
+    void updateGeometry  (const int centerX_, const int centerY_, const double feather_, const double degree_, const int fullWidth=-1, const int fullHeight=-1);
```

Comment:
Should we resolve the name collision of the `centerX` parameter with the `centerX` member using explicit `this->centerX` where necessary inside the methods definition?